src/lib.rs

```diff
@@ -55,7 +55,7 @@ mod static_files_handler;
 mod json_body_parser;
 pub mod mimes;
 mod query_string;
-mod urlencoded;
+pub mod urlencoded;
```

Comment:
A little wary on publicly exposing this, what is the requirement? (and can it be separated from this PR)